cartography/intel/aws/__init__.py

```diff
@@ -144,21 +145,35 @@ def _sync_multiple_accounts(
     logger.info("Syncing AWS accounts: %s", ', '.join(accounts.values()))
     organizations.sync(neo4j_session, accounts, sync_tag, common_job_parameters)
 
+    failed_account_ids = []
+    exception_tracebacks = []
+
     for profile_name, account_id in accounts.items():
         logger.info("Syncing AWS account with ID '%s' using configured profile '%s'.", account_id, profile_name)
         common_job_parameters["AWS_ID"] = account_id
         boto3_session = boto3.Session(profile_name=profile_name)
 
         _autodiscover_accounts(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters)
 
-        _sync_one_account(
-            neo4j_session,
-            boto3_session,
-            account_id,
-            sync_tag,
-            common_job_parameters,
-            aws_requested_syncs=aws_requested_syncs,  # Could be replaced later with per-account requested syncs
-        )
+        try:
+            _sync_one_account(
+                neo4j_session,
+                boto3_session,
+                account_id,
+                sync_tag,
+                common_job_parameters,
+                aws_requested_syncs=aws_requested_syncs,  # Could be replaced later with per-account requested syncs
+            )
+        except Exception as e:
+            failed_account_ids.append(account_id)
+            exception_traceback = traceback.TracebackException.from_exception(e)
+            traceback_string = ''.join(exception_traceback.format())
+            exception_tracebacks.append(traceback_string)
+            continue
```

Comment:
Talked with @ramonpetgrave64 and it might be good to make this behavior configurable:

maybe something like
```
        if config.aws_best_effort_mode: # 
            # append the traceback and continue
            # ...
        else:
            raise
```

What do you think @erickduran?

Reply:
Yeah I like that, on it

Reply:
Just added this, note that  following effects when using `aws_best_effort_mode` and there is at least one error:
- will still try to execute the `run_analysis` functions after `_sync_multiple_accounts` finishes
- will not perform `run_cleanup_job`

Let me know your thoughts

Reply:
Very good callouts. I like how you're quickly getting familiar with our technical  debt 😂 .

> will not perform run_cleanup_job

This is ok.

> will still try to execute the run_analysis functions after _sync_multiple_accounts finishes

I have an idea: What are your thoughts on having `_sync_multiple_accounts()` return True if all accounts succeeded and False otherwise? Then in `start_aws_ingestion()` we could do something like

```python
sync_successful = _sync_multiple_accounts()
...
if sync_successful:
  # run analysis jobs
```